packages/labs/src/components/select/multiSelect.tsx

```diff
@@ -79,7 +79,17 @@ export class MultiSelect<T> extends React.Component<IMultiSelectProps<T>, IMulti
     private input: HTMLInputElement;
     private queryList: QueryList<T>;
     private refHandlers = {
-        input: (ref: HTMLInputElement) => (this.input = ref),
+        input: (ref: HTMLInputElement) => {
+            this.input = ref;
+
+            const { tagInputProps = {} } = this.props;
+            const inputProps: HTMLInputProps = tagInputProps.inputProps || {};
```

Comment:
`const { tagInput: { inputProps = {} } } = this.props` should do the trick

Reply:
But if tagInputProps is undefined, will this line brake?

Reply:
truth, it will. then do the same thing twice. 
`= {}` is the new `|| {}` (and so much more typesafe)
```
const { tagInputProps = {} } = this.props;
const { inputProps = {} } = tagInputProps;
```